### PR TITLE
[master] Windows 10 compatibility fixes

### PR DESCRIPTION
**Pull request by stranger80**
Pull request to aggregate the Windows 10 compatibility changes, introduces as a result of 'Alpha testing' round of yagna.

- The testnet faucet request was failing on WIn 10 machines. It appears the Actix Web Client is useless when trying to call a URL including a DNS name, which requires DNS resolution, on Windows machine with VM virtual switches configured. It is a known issue, which noone seems to be addressing (https://github.com/actix/actix-web/issues/858) - AWC uses non-default configuration of trust-dns-resolver package, and it seems impossible to easily override it.
Therefore AWC client has been replaced with a 'hyper' HTTP client.

### CODE REVIEW

File: core/payment-driver/gnt/src/gnt/faucet.rs [hunk 1]
```diff
@@ -24,18 +28,37 @@ impl EthFaucetConfig {
 
     pub async fn request_eth(&self, address: Address) -> GNTDriverResult<()> {
         log::debug!("request eth");
-        let client = awc::Client::new();
+        let client = hyper::Client::new();
```
Comment (bidzyyys) [inline core/payment-driver/gnt/src/gnt/faucet.rs:31]: Please remove `awc` crate from gnt driver if is redundant.

**Review by bidzyyys — APPROVED** [review]

**Comment by stranger80** [comment]
Removed awc dependency from GNT driver. Pls approve.

**Review by bidzyyys — COMMENTED** [review]
I will approve if tests pass

**Comment by stranger80** [comment]
Tests passed now :)

**Review by bidzyyys — APPROVED** [review]
👍

**Review by tworec — APPROVED** [review]

**Pull request merged**